msal/application.py

```diff
@@ -97,6 +97,7 @@ def __init__(
         """Create an instance of application.
 
         :param client_id: Your app has a client_id after you register it on AAD.
+
         :param client_credential:
```

Comment:
Consider a type hint here, e.g. `:param str client_credential:` Tools like pycharm pick up on these hints and make it easier to determine the value to pass in.